### PR TITLE
#37 response schedule

### LINKED ISSUE #37: US-37: As a user, I would like to notify my manager user if I accept or reject a schedule he/she set for me

### PR DESCRIPTION
**Pull request by KhangZhengNg**
### Related Issues
#37 

### Description
Add the top app bar to ScheduleViewRespondActivity
Unit and UI testing on ScheduleViewRespondActivity and its view model

### How to check PR
Run ScheduleViewRespondActivity to check the top app bar
Run the ScheduleViewRespondViewModelTest and ScheduleViewRespondActivityTest

### CODE REVIEW

**Comment by refatK** [comment]
@KhangZhengNg 

The build is failing, but it's a pretty simple fix, just do this change:
https://github.com/Landry333/Big-Owl/commit/1c15f384a608cc4566fc9ca0c4a784d032bab6d2

File: app/src/androidTest/java/com/example/bigowlapp/activity/ScheduleViewRespondActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+    @Test
+    public void respondScheduleTest() {
+        // accept schedule
+        when(mockScheduleViewRespondViewModel.getCurrentUserNewResponse()).thenReturn(new UserScheduleResponse(Response.ACCEPT, Timestamp.now()));
+        when(mockScheduleViewRespondViewModel.getUserScheduleResponse()).thenReturn(new UserScheduleResponse(Response.ACCEPT, Timestamp.now()));
+        onView(withId(R.id.button_accept)).perform(click());
+        verify(mockScheduleViewRespondViewModel, times(1)).isOneMinuteAfterLastResponse();
+        verify(mockScheduleViewRespondViewModel, times(1)).respondSchedule(testSchedule.getuId(), Response.ACCEPT);
+        verify(mockScheduleViewRespondViewModel, times(1)).notifySupervisorScheduleResponse();
```
Comment (ShawnGuerra-Bautista) [inline app/src/androidTest/java/com/example/bigowlapp/activity/ScheduleViewRespondActivityTest.java:160]: The test ran, but failed on this line.

**Review by ShawnGuerra-Bautista — CHANGES_REQUESTED** [review]
Tests ran perfectly, and the coverage is reasonable.
However, the "respondScheduleTest" integrated (UI) test is failing, and should be fixed.

Edit: @refatK notified me that he did some fixes on your tests. This branch can probably help: https://github.com/Landry333/Big-Owl/tree/%2337-response-schedule-with-test-fix.

File: app/src/androidTest/java/com/example/bigowlapp/activity/ScheduleViewRespondActivityTest.java (hunk 1 above)
Comment (KhangZhengNg) [reply app/src/androidTest/java/com/example/bigowlapp/activity/ScheduleViewRespondActivityTest.java:160]: thanks for the review!

**Review by ShawnGuerra-Bautista — APPROVED** [review]

**Review by refatK — APPROVED** [review]
Just ran the app on my end, and works as expected. App bar works, the notifications get generated upon accepting or rejecting, and the coverage is high and there is no quality issues.

Good job!

**Pull request merged**